### DISCUSSION TITLE
What is remaining before Milestone 5.1.0 can be released

### DISCUSSION THREAD

**Discussion by rfair404** [stuttter/ludicrousdb#167] [General]
I have been tracking the [5.1.0](https://github.com/stuttter/ludicrousdb/milestone/3) milestone, as I am hoping that this will repair some of the deprecated warnings that I am seeing in a site running php 8.2. I am curious, what is blocking this release from being published? There are no remaining open issues, that I can see. @JJJ - what is the timeline for tagging the release?

**Comment by JJJ** [comment]
Soon™️

Hopefully this week!

**Reply by rfair404** [reply]
```Soon™️``` 🤣

**Comment by JJJ** [comment]
5.2.0 is out!

**Comment by rfair404** [comment]
Awesome! Thank you due the update and your hard work on maintaining this
plugin!

**Reply by JJJ** [reply]
You're welcome! Happy to do it!
